docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Troubleshooting Server Access
+description: Describes common issues and solutions for access to servers.
+---
+
+This section describes common issues that you might encounter in managing access to servers
+with Teleport and how to work around or resolve them.
+
+## Starting SSH sessions fails
+
+By default, when Teleport connects to a server, the service runs as the user login that starts 
+the SSH session. If starting the session fails, it might be because the file system permissions
+for the Teleport binary don't allow other user logins to connect to the remote server.
+This issue might be caused by systems that prevent newly installed software from being executable 
+by any user.  
+
+### Symptom
+
+This issue results in a disconnected session and an error message similar to the following:
+
+```text
+Failed to launch: fork/exec /proc/self/exe: permission denied.
+Process exited with status 255
+```
+
+### Solution
+
+You should check the permission settings for the `teleport` binary. 
+
+To check the file system permissions on the `teleport` binary:
+
+1. Open a terminal shell on the computer where you have installed the `teleport` service.
+
+1. Determine the location and file system permission of the Teleport binary by running the following command:
+   
+   ```code
+   ls -al $(which teleport)
+   ```
+
+   The command should return output similar to the following:
+   
+   ```text
+   -rwxr-xr-x  1 root  wheel  531849504 Aug 30 18:32 /usr/local/bin/teleport
+   ```
+   
+   If you don't see the permission that allows other users to read and execute (-rwxr-x**r-x**), 
+   you should update the permissions. For example:
+
+   ```code
+   chmod o+rx teleport
```

Comment:
In the previous step, we suggest that we need all 3 execute bits to be set (note the three `x` characters in `rwxr-x**r-x**`).

`chmod o+rx` is only going to set the third and final one. To be consistent with what we said before, we probably want to say `chmod go+rx`.